tests/wicked/basic/sut/t08_setup_second_card.pm

```diff
@@ -16,13 +16,14 @@ use base 'wickedbase';
 use strict;
 use warnings;
 use testapi;
-
+use lockapi;
 
 sub run {
     my ($self, $ctx) = @_;
     my $cfg_ifc1 = '/etc/sysconfig/network/ifcfg-' . $ctx->iface();
     my $cfg_ifc2 = '/etc/sysconfig/network/ifcfg-' . $ctx->iface2();
     record_info('Info', 'Set up a second card');
+    mutex_wait('DHCP_READY');
```

Comment:
please move it just before line 30 so we wait exactly before where we need DHCP to be ready

Reply:
done